src/usexp/lexer0.mli

```diff
@@ -0,0 +1,44 @@
+module Token : sig
```

Comment:
What about calling this file lexer_common.mli? I find 0 suffixed names to not be very descriptive...

Reply:
I try to avoid filler words like common, misc etc. `Lexer_shared` or `Lexerlib`?

Not sure if this is an improvement...

Reply:
`Lexer_shared` looks good. I personally think it's an improvement as the `0` suffix makes me think of something used to break a circular dependency. At least that's how we use it inside the Jane Street code base.